crates/proto/src/error.rs

```diff
@@ -17,6 +17,9 @@ use alloc::string::ToString;
 use alloc::sync::Arc;
 use alloc::vec::Vec;
 use core::fmt;
+use core::num::ParseIntError;
+#[cfg(any(feature = "__https", feature = "__h3"))]
+use http::header::ToStrError;
```

Comment:
It seems odd that this 3rd party crate import is between `core` and `std` imports. Shouldn't it be after both, potentially with separating whitespace?